### PR TITLE
ProjectWidget RTL

### PR DESCRIPTION
**Pull request by Erin007**
This PR includes styling changes that better support RTL for the `ProjectWidget`. 
 
BEFORE: 
rtl projectwidget before

AFTER: 
rtl projectwidget after

### CODE REVIEW

**Comment by Erin007** [comment]
cc: @tanyaparker

File: apps/src/templates/projects/ProjectWidgetWithData.jsx [hunk 1]
```diff
@@ -30,11 +31,14 @@ const ProjectWidgetWithData = React.createClass({
   },
 
   render() {
+    const { isRtl } = this.props;
+
```
Comment (caleybrock) [inline apps/src/templates/projects/ProjectWidgetWithData.jsx:35]: nit: you're only using this in one place, it doesn't feel necessary to make it a constant.
Count just say `isRtl={this.props.isRtl}`

File: apps/src/templates/projects/ProjectWidgetWithData.jsx [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
 import ProjectWidget from '@cdo/apps/templates/projects/ProjectWidget';
 import $ from 'jquery';
 
 const ProjectWidgetWithData = React.createClass({
   propTypes: {
-    projectTypes: React.PropTypes.arrayOf(React.PropTypes.string),
-    projectList: React.PropTypes.array
+    projectTypes: PropTypes.arrayOf(React.PropTypes.string),
+    projectList: PropTypes.array,
+    isRtl: PropTypes.bool.isRequired
```
Comment (caleybrock) [inline apps/src/templates/projects/ProjectWidgetWithData.jsx]: This component also gets used in StageExtras. Can we either update it their (if CSF has a rtl mode) or make this prop not required (default to false)?

File: apps/src/templates/projects/PersonalRecentProjects.jsx [hunk 3]
```diff
@@ -32,17 +35,21 @@ const styles = {
...
 const ProjectCardGrid = React.createClass({
   propTypes: {
-    projectList: PropTypes.arrayOf(personalProjectDataPropType).isRequired
+    projectList: PropTypes.arrayOf(personalProjectDataPropType).isRequired,
+    isRtl: PropTypes.bool.isRequired
   },
 
   render() {
+    const { isRtl } = this.props;
+    const cardPaddingStyle = isRtl ? styles.cardPaddingRtl : styles.cardPadding;
```
Comment (caleybrock) [inline apps/src/templates/projects/PersonalRecentProjects.jsx:44]: again, no need to make this a constant if it's only being used on the next line.

**Comment by caleybrock** [comment]
Somehow I didn't know how these flags were being used - this looks good. Thanks for fixing.

File: apps/src/templates/projects/ProjectWidgetWithData.jsx (hunk 2 above)
Comment (Erin007) [reply apps/src/templates/projects/ProjectWidgetWithData.jsx]: I set isRtl to false in StageExtras.

Comment (caleybrock) [reply apps/src/templates/projects/ProjectWidgetWithData.jsx]: In a different PR? Don't see it here so wondering if I'm missing something.

File: apps/src/templates/studioHomepages/TeacherHomepage.jsx [hunk 4]
```diff
@@ -78,7 +78,9 @@ export default class TeacherHomepage extends React.Component {
         <TeacherResources
           isRtl={isRtl}
         />
-        <ProjectWidgetWithData/>
+        <ProjectWidgetWithData
```
Comment (breville) [inline apps/src/templates/studioHomepages/TeacherHomepage.jsx]: nit: This can be a single line since it's a single param.

**Comment by caleybrock** [comment]
👍 yep lgtm as long as stage extras looks good.

File: apps/src/templates/projects/ProjectWidgetWithData.jsx (hunk 2 above)
Comment (Erin007) [reply apps/src/templates/projects/ProjectWidgetWithData.jsx]: I changed my mind and just made the prop not required.

**Pull request merged**